warp10/src/main/java/io/warp10/script/WarpScriptLib.java

```diff
@@ -857,7 +857,8 @@
     functions.put("mapper.exp", new MapperExp.Builder("mapper.exp"));
     functions.put("mapper.log", new MapperLog.Builder("mapper.log"));
     functions.put("mapper.min.x", new MapperMinX.Builder("mapper.min.x"));          
-    functions.put("mapper.max.x", new MapperMaxX.Builder("mapper.max.x"));          
+    functions.put("mapper.max.x", new MapperMaxX.Builder("mapper.max.x"));  
+    functions.put("mapper.todouble.from.language", new MapperToDouble.Builder("mapper.todouble.from.language"));        
```

Comment:
Since the 'language' only has a use for string values, maybe this should be made a separate function which applies to strings with a more meaningful name.